compositor_common/src/util.rs

```diff
@@ -95,6 +95,57 @@ pub enum ColorParseError {
     HexNumberParseError(#[from] ParseIntError),
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, SerializeDisplay, DeserializeFromStr)]
+pub enum Coord {
+    Pixel(i32),
+    Percent(i32),
+}
+
+impl Display for Coord {
```

Comment:
that implementation should be symetrical if you are using them to implement serialization and deserialization
consider writing a test case for that